torch/fx/proxy.py

```diff
@@ -83,6 +83,9 @@ def __exit__(self, *args):
         return
 
 
+_COPY_META_FIELDS = ["nn_module_stack", "source_fn", "original_aten", "recompute", "from_node"]
```

Comment:
So right now we are overloading single entry point of export for quant. WIth longer term plan of moving to 1) pre_dispatch_export and 2) aten_export, I am wondering if it would make more sense to let passes add  what metdata they want to presreve.

e.g. 
m = pre_dpsatch_export(m)
quantized_m = quantize..(m).. # this step adds what metadata quantization wants to preserved. and it conveys this by embeding this info in quantized_m?